src/Build/BackEnd/BuildManager/BuildRequestData.cs

```diff
@@ -75,6 +75,11 @@ public enum BuildRequestDataFlags
         /// This is especially useful during a restore since some imports might come from packages that haven't been restored yet.
         /// </summary>
         IgnoreMissingEmptyAndInvalidImports = 1 << 6,
+
+        /// <summary>
+        /// Turns on logic for graph based isolated project builds
+        /// </summary>
+        IsolatedProjectBuild = 1 << 7
```

Comment:
I don't see where this is ever set. Is that intended to be in this PR, or a future PR which wires up the command-line flag to this?

Reply:
It's set in a test :)
Future PRs will pipe the end to end scenario.